inscriptions/ISic003700.xml

```diff
@@ -191,73 +196,30 @@
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">ἀγορασία Π</supplied>αλλαδίου τοῦ
-                    <lb n="2"/>
-                    <supplied reason="lost">συμβίου τ</supplied>ῆς Πελαγίας· ἐ<supplied reason="lost">γέ</supplied>-
-                    <lb n="3"/>
-                    <supplied reason="lost">νετο Πελαγία</supplied>
-                    τῇ πρὸ ιαʹ καλ
-                    <supplied reason="lost">ανδ(ῶν)</supplied>
-                    <lb n="4"/>
-                    <supplied reason="lost">— — — — — —</supplied>․, τελευτᾷ δ
-                    <supplied reason="lost">ὲ</supplied>
-                    <lb n="5"/>
-                    <supplied reason="lost">— — — — — Σε</supplied>πτεμβρίων.
-                    <lb n="6"/>
-                    <supplied reason="lost">ἐνθάδε κῖτ</supplied>ε μόνανδρο<supplied reason="lost">ς</supplied>.
-                    <lb n="7"/>
+                    <lb n="1"/><supplied reason="lost">ἀγορασία</supplied> <name><supplied reason="lost">Π</supplied>αλλαδίου</name> τοῦ
+                    <lb n="2"/><supplied reason="lost">συμβίου</supplied> <supplied reason="lost">τ</supplied>ῆς <name>Πελαγίας</name> ἐ<supplied reason="lost">γέ</supplied>
+                    <lb n="3" break="no"/><supplied reason="lost">νετο</supplied> <supplied reason="lost"><name>Πελαγία</name></supplied> τῇ <hi rend="ligature">πρ</hi>ὸ <num value="11">ια</num> καλ<supplied reason="lost">ανδων</supplied>
+                    <lb n="4"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">τελε</supplied>υτᾶ δ<supplied reason="lost">ὲ</supplied>
+                    <lb n="5"/><gap reason="lost" extent="unknown" unit="character"/> <supplied reason="lost">Σε</supplied>πτεμβρίων
+                    <lb n="6"/><supplied reason="lost">ἐνθάδε</supplied> <supplied reason="lost">κῖτ</supplied>ε μόνανδρο<supplied reason="lost">ς</supplied>                    
                 </ab>
             </div>
             <div type="apparatus">
-                <p><!--external apparatus criticus (if applicable)--></p>
+                <listApp>
+                    <app><note>Text: Ferrua 1989</note></app>
+                    <app loc="line 1"><note>IG: [--- Π]α[λλ]αδίου; CIG: [Μνημεῖον Γενν]αδίου</note></app>
+                    <app loc="line 5"><note>IG and CIG: [Σε]π[τ]εμβρίω[ν] </note></app>
+                    <app loc="line 6"><note>CIG: --- ἐμὸν ἃνδρ[α]</note></app>
+                </listApp>
             </div>
             <div type="translation">
-                <p><!--translation(s) - add @xml:lang for 'en' or 'it' to div--></p>
+                <p>The purchase of(?) Palladios, the spouse of Pelagia. Pelagia having passed 11 [days] before the Kalends of [...], having died [...] of September, here she, who had one husband, lies</p> <!-- unsure about this translation -->
```

Comment:
i think it's fine - agorasia is a standard formulation for the purchase of the loculus /tomb